### DISCUSSION TITLE
Historical data other extra service

### DISCUSSION THREAD

**Discussion by pawelosin** [RavenSystem/esp-homekit-devices#1564] [Q&A] [answered]
If I’am using other extra service, historical data does not appear in HAA Manager. For example power measure. My json:
{"c":{"o":1,"l":1,"n":"pralnia-pralka","ntp":"pool.ntp.org","ct":7,"b":[{"g":3,"t":5}]},"a":[{"t":2,"0":{"r":[{"g":14},{"g":13,"v":1}]},"1":{"r":[{"g":14,"v":1},{"g":13}]},"s":5,"b":[{"g":3}],"es":[{"t":6},{"t":75, "es": [{"t": 95,"h": [ 1, 0, 5 ]}],"j":5,"dt":[4,5,12],"vf":0.16,"cf":0.1,"pf":1.706 ,"y2":[{"v":0,"r":1,"0":{"m":[[2,1]]}},{"v":2.05,"r":1,"0":{"m":[[2,0]]}}]}]}]}
Any suggestions?

**Comment by RavenSystem** [comment, marked as answer]
You can not declare an "es" array into other "es" array. Declare all extra services into same "es" array.
